apps/meteor/app/livechat/server/startup.ts

```diff
@@ -21,7 +22,11 @@ import './roomAccessValidator.internalService';
 const logger = new Logger('LivechatStartup');
 
 Meteor.startup(async () => {
-	roomCoordinator.setRoomFind('l', (_id) => LivechatRooms.findOneById(_id));
+	roomCoordinator.setRoomFind('l', async (_id) => {
+		const room = await LivechatRooms.findOneById(_id);
+
+		return (await maybeMigrateLivechatRoom(room)) ?? undefined;
```

Comment:
I noticed that `LivechatRooms.findOneById` returns `IOmnichannelRoom | null`, same for `maybeMigrateLivechatRoom`. Are we intentionally changing the fallback to `undefined` here?

Reply:
`roomFind?: (rid: string) => Promise<IRoom | undefined> | Promise<IOmnichannelRoom | null> | IRoom | undefined;`

:(

But this was the last place still calling maybeMigrateLivechatRoom, so I changed it to use IOmnichannelRoom instead of IRoom.